src/RudderClient.ts

```diff
@@ -193,19 +193,26 @@ async function group(groupId: string, traits: Object | null = null) {
     logWarn("group: 'groupId' must be a string");
     return;
   }
-  logWarn("group: Method not supported");
+  bridge.group(groupId, traits, options)
 }
 
-async function alias(previousId: string, userId: string) {
-  if (previousId == undefined) {
-    logWarn("alias: Mandatory field 'previousId' missing");
+async function alias(previousId: string, userId: string): Promise<void>; 
+async function alias(newId: string, options: Object | null = null) {
+  if (newId == undefined) {
+    logWarn("alias: Mandatory field 'newId' missing");
     return;
   }
-  if (typeof previousId != "string") {
-    logWarn("alias: 'previousID' must be a string");
+  if (typeof newId != "string") {
+    logWarn("alias: 'newId' must be a string");
     return;
   }
-  logWarn("alias: Method not supported");
+  if (typeof options == "string") {
```

Comment:
I don't think we need to check if `options` passed is a `string` type

In this case if both `newId` and and `options` are sent as a string, as following:
```
alias("newId1", "optionsString");
```

then the final resultant call would be 
```
bridge.alias("optionsString");
```
which is not an expected behavior

Reply:
We have to for backward compatibility.

Reply:
Got it